content/news/2023-01-beacon-integration/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Integration of Beacon into Galaxy'
+date: "2023-01-24"
+tease: "We have added an integration of Beacon into galaxy"
+hide_tease: true
+tags: [galaxy, collaboration, UseCase]
+subsites: [global]
+---
+
+Science is most efficient when researchers share their data and results, so others can re-use them. Some datasets, however, cannot be easily shared because they contain sensitive information about, for example, a patient. Is there a way to share information among scientists without exposing sensitive patient data? The answer to that question for human genome variation data is: yes. 
+The [Beacon network](https://beacon-network.org/#/) enables secure querying of genomic variants without the risk of exposing sensible data. The data owner is responsible for the data and also for their Beacon server in the network. In this way the data never leaves the data owner. A Beacon only contains the raw variants, think about it as some marked coordinates on the chromosome.
+A researcher interested in the genomic position 32,936,732 on chromosome 13 can ask the entire Beacon network if something about this position is known. In the simplest case the researcher gets a “Yes” or “No” answer. In a positive case, the researcher gets a contact address to the Beacon server, or data owner, so that both parties can negotiate the exchange of more detailed data. 
+In that sense Beacon facilitates the first-contact and speeds up knowledge exchange dramatically - often benefiting the patients in clinical care.
+
+
+![https://beacon-project.io/](./Beacon-v1.png)
```

Comment:
It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details. *This comment may apply to more images, please check the others as well.*

Reply:
I have to admit that I didnt know that those were alt texts. Hopefully the texts I just added are a bit more descriptive of whats visible in the images :sweat_smile: :.

Thank you for the insight!